src/inputs/inputs-controller.ts

```diff
@@ -356,11 +424,11 @@ export class InputsController {
    *
    * @param event The keyboard event.
    */
-  keyboardKeyDown(event): void {
+  private keyboardKeyDown(event): void {
     this.lastSource = "keyboard";
     this.ensureKeyboardIsInit();
-    const buttonDown = getButtonWithKeycode(this.getActiveConfig(Device.KEYBOARD), event.keyCode);
-    if (buttonDown !== undefined) {
+    const buttonDown = getButtonWithKeycode(this.getActiveConfig(Device.KEYBOARD)!, event.keyCode);
```

Comment:
```suggestion
    const config = this.getActiveConfig(Device.KEYBOARD);
    if (!config) {
      return; // The keyboard isn't setup, ignore the input
    }
    const buttonDown = getButtonWithKeycode(config, event.keyCode);
```
This was added to `keyboardKeyUp` but not here.

Reply:
ensureKeyboardIsInit initializes the config in onKeyboardKeyDown, so there shouldn't be a need to double check the config there
In onKeyboardKeyUp I chose to check the config rather than initialize the keyboard because I don't want the input to be interpreted if no down input was received before.
An 'up' event being received before any 'down' event is what caused the crash in #1094